.github/workflows/linux.yml

```diff
@@ -0,0 +1,59 @@
+name: Linux
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        default: "102"
+        required: true
+        type: choice
+        options: ["102", "115"]
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+
+  
+env:
+  SCCACHE_GHA_ENABLED: "true"
+  CCACHE: "sccache"
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
+      with:
+        # remove the ones that are set to true
+        tool-cache: false
+        android: true
+        dotnet: true
+        haskell: true
+        large-packages: false
+        docker-images: true
+        swap-storage: false
+    - name: init env
+      run: |
+        source ${{ inputs.VERSION }}/${{ inputs.VERSION }}.sh
+        rustup override set $RUST_VER
+        sudo apt update
+        sudo apt install libdbusmenu-glib-dev libdbusmenu-gtk3-dev libdbusmenu-gtk-dev -y
```

Comment:
Already counted on.

Reply:
Good. Maybe the extra ones fix my compile.